console/billing/additional-content/understanding-savings-plans.mdx

```diff
@@ -0,0 +1,114 @@
+
+Currently, Scaleway only offers Savings Plans for compute resources.
+
+### Commitment amount
+
+The monthly commitment amount for a Savings Plan must be at least €1000. The amount you specify will determine in which discount bracket your plan falls.
+
+### Duration
+
+The commitment period can last 12, 24, or 36 months. The longer the commitment, the higher the discount offered.
+
+### Payment options
+
+Payments are made on a monthly basis as part of your regular billing cycle.
+
+### Discounts
+
+Savings Plan discounts are applied automatically to the eligible services.
+
+At Scaleway, the discounts are applied in the following order:
+    - Free tiers or free trials - Savings Plan discounts will apply after any eligible free tier or trial credits have been used. This ensures that you maximize your free usage before the Savings Plan discount is applied.
+    - Savings plans, and
+    - Discounts (vouchers or credits) - Any applicable vouchers or credits are used and applied after the Savings Plan discount.
+
+Understanding this order helps you optimize your savings and ensure that you make the most of the available discounts.
+
+### Cancellations and exchanges
+
+Savings plans cannot be canceled or exchanged.
+
+If you need to request a change to your savings plan, [contact the sales team](https://www.scaleway.com/en/contact-sales/).
+
+We do not make refunds for savings plans that have already been invoiced.
+
+<Message type="important">
+  If you have early access to the Savings Plans, you can delete your Savings Plan at any time. The deletion of a savings plan will take effect at the end of the month, regardless of the date the deletion request is made.
+</Message>
+
+### Taxes and currency
+
+The total cost displayed for the savings plan includes applicable taxes, and these will be itemized on your monthly invoice.
+
+Currently, we invoice all our customers in euros (EUR). If your primary currency is different, our Payment Service Providers (PSPs) apply the prevailing exchange rate without any additional currency conversion fees on your part.
+
+## How to purchase a Savings Plan
+
+Follow the instructions on the [How to purchase a Savings Plan](/console/billing/how-to/purchase-savings-plan) documentation plan to do so.
+
+## Compute Savings Plan
+
+There is currently one available Savings Plan type: the Compute Savings Plan.
+
+The **Compute Savings Plan** can be used with the following resources, simultaneously and across all regions:
+
+    - Instances 
+      - Cost-optimized (COPARM1, PLAY2, PRO2)
+      - Production optimized (ENT1, POP2)
+      - Workload optmized (POP2 HC) 
+    - Dynamic IPs, IPv6, Reserved IPs
+    - Block Storage SSD
+    - Block Storage volumes
+    - Block Storage Low Latency volumes
+
+The following resources are **not** covered by the Savings Plan discount:
+
+    - Instances H100, RENDER, L40, L4, DEV1, GP1, START1, X64, POP-WIN
+    - Instance and Block Storage snapshots 
+    - Local storage 
+
```

Comment:
Maybe here we can add a note indicating that other scaleway products are not included, and that the list of products included or not in the savings plans may change regularly.